src/pybind/mgr/cephadm/module.py

```diff
@@ -536,7 +544,8 @@ def config_notify(self):
         self.event.set()
 
     def notify(self, notify_type, notify_id):
-        pass
+        if notify_type == "mon_map":
```

Comment:
Is there any kind of documentation which type of `notify_type`s there  are?

Reply:
https://docs.ceph.com/docs/master/mgr/modules/#receiving-notifications